doc/source/batch-inference/preprocessing.rst

```diff
@@ -0,0 +1,118 @@
+.. _inference_preprocessing:
+
+Preprocessing Data
+==================
+
+After loading in a Dataset, the input the input data often needs to be preprocessed prior to inference. This may include cropping or resizing images, or tokenizing raw text.
+
+With :ref:`Ray Data <datasets>`, you can define user-defined functions (UDFs) that transform batches of your data. Applying these UDFs via :meth:`ds.map_batches() <ray.data.Dataset.map_batches>` will output a new, transformed Dataset.
+
+.. tip::
+    
+    For the full suite of Dataset transformations, read :ref:`transforming_datasets`.
+
+
+Writing batch UDFs
+------------------
```

Comment:
Why are we calling them UDF? That seems like Spark terminology. Why don't we call them stateful model classes or something ?

Reply:
Agree... we could use this phrasing instead: `With Ray Data, you can define custom models (implemented as ordinary Python classes) that transform batches of your data.`.

Reply:
I think this is common lingo in this context. My worry is rather that "UDF" and other acronyms might look intimidating to new users